concepts/loops/introduction.md

```diff
@@ -0,0 +1,3 @@
+# Introduction
+
+Loops enable repetition of blocks of code. There are three types of loops in Rust: the `for` loop that enables looping over elements of an iterator or a collection (technically, values of any type that implements the `IntoIterator` trait), the `while` loop that repeates a block of code as long as a condition is satisfied and the `loop` loop that indefinitely repeats code until a `break` is invoked explicitly.
```

Comment:
At first I thought you  must have this comma I have added, but I have just read https://en.wikipedia.org/wiki/Serial_comma and I now see that it is not necessary. In that case, I'll approve anyway; feel free to have the comma or not.

```suggestion
Loops enable repetition of blocks of code. There are three types of loops in Rust: the `for` loop that enables looping over elements of an iterator or a collection (technically, values of any type that implements the `IntoIterator` trait), the `while` loop that repeates a block of code as long as a condition is satisfied, and the `loop` loop that indefinitely repeats code until a `break` is invoked explicitly.
```